src/components/tree/tree-item.ts

```diff
@@ -82,6 +88,10 @@ export default class IgcTreeItemComponent extends LitElement {
   @blazorSuppress()
   public wrapper!: HTMLElement;
 
+  /** @private */
+  @query('[role="group"]', true)
+  public group!: HTMLElement;
```

Comment:
Do this need to be a public property at all?

Reply:
This has been addressed.